scissors-sample/src/main/java/com/lyft/android/scissorssample/MainActivity.java

```diff
@@ -103,6 +112,28 @@ public void onPickClicked() {
                 .pickUsing(this, RequestCodes.PICK_IMAGE_FROM_GALLERY);
     }
 
+    @OnClick(R.id.ratio_fab)
+    public void onRatioClicked() {
```

Comment:
What did you have in mind?  Like a toast?  Could do the button contents, but not sure how it would fit in a mini fab.